pyro/distributions/distribution.py

```diff
@@ -166,3 +166,8 @@ def has_rsample_(self, value):
             raise ValueError("Expected value in {False,True}, actual {}".format(value))
         self.has_rsample = value
         return self
+
+    @property
+    def rv(self):
```

Comment:
Could you add a big docstring here with an EXPERIMENTAL warning and example usage, something like
```
"""
EXPERIMENTAL Switch to the Random Variable DSL for transforming random variables.

Example usage::

    # This should be equivalent to an Exponential distribution.
    Uniform(0,1).rv.log().neg().dist

:returns: A :class:`~pyro.contrib.randomvariable.random_variable.RandomVariable`
    object wrapping this distribution.
:rtype: ~pyro.contrib.randomvariable.random_variable.RandomVariable
"""
```